### DISCUSSION TITLE
UI5 Test Recorder support for WDI5

### DISCUSSION THREAD

**Discussion by ph-alsvik** [ui5-community/wdi5#187] [Q&A] [answered]
Hi guys! 

Do you know if the [UI5 Test Recorder](https://openui5nightly.hana.ondemand.com/index.html#/topic/dac59fadd5f9419d986f74ba602c6d29) will also support wdi5 control selector syntax to make it easier to find control selectors? 

Also, do you see the test recorder-tool as being an integral part of developing tests that run with wdi5?

**Comment by Siolto** [comment, marked as answer]
Hi Peder,

we are already working on another, so called, _dialect_ for the Test Recorder. There is already an [issue](https://github.com/js-soft/wdi5/issues/149) for this. 

The Test Recorder is a useful tool to build good and stable running tests. It is a good addition to identify the controls easier and faster.
As with any tool, you should not assume that it will generate the best selectors in every specific case. You should always check the generated selectors to see if there is a better alternative.

I personally also use the test recorder regularly in the development process of the tests. Especially with Fiori elements applications, where you can't find the right matcher immediately, the test recorder helps me a lot.

Best regards
Simon
